src/css/components/_captions-settings.scss

```diff
@@ -101,5 +97,5 @@
 }
 
 .vjs-track-settings-controls .vjs-default-button {
-  margin-right: 1em;
+  margin: 1em;
```

Comment:
with this change, the controls end up being 20px+1em from the button of the dialog, if we keep it as 1em, they'll only be the 20px from the bottom and look more consistent with the other controls.
The 20px come from the padding of the dialog itself.

Reply:
So the reason why this was changed, was that when the player is small the reset/done buttons touch the very bottom of the captions-setting modal. Maybe we can just add the margin-bottom at small sizes, and make this margin-right again.'

Reply:
Ah, didn't notice that. Looking into it, it seems that when it overflows, the "modal" itself doesn't resize, so, we don't get the padding from it. It looks like what we want to do is set `overflow: scroll` on the modal itself when we know it'll scroll, or probably just always set overflow scroll for this modal.
Tried it out and seems to work.

Reply:
after testing this out, it worked but only on chrome, on firefox it actually made the settings go off the modal, and they could not be scrolled to.